jax/scipy/ndimage.py

```diff
@@ -37,7 +37,7 @@
 
 
 def _nearest_indices_and_weights(coordinate):
-  index = jnp.around(coordinate).astype(jnp.int32)
+  index = jnp.floor(coordinate + .5).astype(jnp.int32)
```

Comment:
Could you define a helper function for this, with a name like `_round_half_up`? That would make the intent of this code more obvious (and maybe be useful below as well).

Reply:
This is a rabbit hole. Scipy rounding is symmetric which means -1.5 and 1.5 get rounded to -2 and 2 respectively. but `np.floor(-1.5 + .5) == -1`

Reply:
done